### PR TITLE
Add an example to documentation for reserved ORM field names

### PR DESCRIPTION
**Pull request by lukapeschke**
Adding a documentation example as discussed in https://github.com/samuelcolvin/pydantic/issues/1463

### CODE REVIEW

File: docs/examples/models_orm_mode_reserved_name.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import typing
+
+from pydantic import BaseModel, Field
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+
+class MyModel(BaseModel):
+    metadata: typing.Dict[str, str] = Field(alias="metadata_")
```
Comment (samuelcolvin) [inline docs/examples/models_orm_mode_reserved_name.py:8]: single quotes please.

Comment (StephenBrown2) [reply docs/examples/models_orm_mode_reserved_name.py:8]: Would you mind a PR to add flake8 linting to `docs/examples`, so you don't need to make comments like these?

Comment (samuelcolvin) [reply docs/examples/models_orm_mode_reserved_name.py:8]: That would be great, but I think you might need to disable many error codes. The many one is that currently docs examples use one blank line everywhere, not two. I'd be happy to change that in all the examples.

Comment (StephenBrown2) [reply docs/examples/models_orm_mode_reserved_name.py:8]: Here ya go! https://github.com/samuelcolvin/pydantic/pull/1530

**Comment by lukapeschke** [comment]
@samuelcolvin thanks for the review! I've updated the example and rebased on master

**Comment by samuelcolvin** [comment]
awesome, thank you.

**Pull request merged**